SAMLPortal/wwwroot/js/site.js

```diff
@@ -2,3 +2,47 @@
 // for details on configuring this project to bundle and minify static web assets.
 
 // Write your JavaScript code.
+function getResultsFromFilters()
+{
+	let uri = "/api/setup/getResultsFromFilters";
+	let results = [];
+	let testUsername = document.getElementById("testUsername").value;
+
+	fetch(uri, {
+		method: "POST",
+		headers: {
+			'Content-Type' : 'application/json'
+		},
+		body: JSON.stringify(testUsername)
+	})
+	.then(response => response.json())
+	.then((response) => {
+		console.log(response);
+		let values = response['value']
+
+		if(Object.keys(values).length > 0)
+		{
+			let resultsDiv = document.getElementById("resultsDiv");
+			resultsDiv.innerHTML = "";
+			
+			let users = values["0"];
+			let admins = values["1"];
+
+			resultsDiv.innerHTML += "<p>Results of User filter :</p>";
+
+			for (var key in users)
+			{
+				var value = values[key];
+				resultsDiv.innerHTML += "<div class='alert alert-primary' role='alert'>" + value + "</div>"
+			}
+
+			resultsDiv.innerHTML += "<p>Results of Administrators filter :</p>"
+
+			for (var key in admins)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.](https://app.codacy.com/manual/zegorax/SAMLPortal/pullRequest?prid=5343548)